### ISSUE TITLE
Compile error if Json type is out of namespace.

### ISSUE THREAD

**Issue by stjeong** [trampster/JsonSrcGen#58]
For example, 

```
using System;
using JsonSrcGen;

/*
namespace ConsoleApp1
{
*/
    [Json]
    public class MyType
    {
        [JsonName("my_name")]
        public string MyProperty { get; set; }

        [JsonIgnore]
        public string IgnoredProperty { get; set; }
    }
// }
```

with above code, lots of compilation error occurred like these,

```
Error CS0501 '<invalid-global-code>.<invalid-global-code>(value, builder)' must declare a body because it is not marked abstract, extern, or partial
Error CS0501 'JsonConverter.ToJson(global)' must declare a body because it is not marked abstract, extern, or partial
Error CS0116 A namespace cannot directly contain members such as fields or methods
Error CS1520 Method must have a return type
...[omitted for brevity]...
```

**Comment by trampster** [comment]
Thanks for you bug report.

I have been able to reproduce the issue. Doesn't look like it should be to hard to fix.

**Comment by trampster** [comment]
I've committed a fix. Will be included in the next 1.1.0 alpha release.
